README.md

```diff
@@ -16,31 +16,31 @@ Fira Code is an extension of the Fira Mono font containing a set of ligatures fo
 
 ### Terminal support
 
-Work | Don’t work
+Works | Doesn’t work
```

Comment:
This is grammatically confusing.  `Work` and `Don't Work` are the plural forms, `works` and `doesn't` are singular forms.